pkgs/peerswap/default.nix

```diff
@@ -0,0 +1,26 @@
+{ pkgs, lib, fetchFromGitHub, buildGoModule, goSubPackages}:
+
+buildGoModule rec {
+  pname = "peerswap-lnd";
+  version = "0.2.0-beta";
+
+  src = fetchFromGitHub {
+      repo = "peerswap";
+      owner = "sputn1ck";
```

Comment:
should this not fetch the official repo at https://github.com/ElementsProject/peerswap/ ?